### ISSUE TITLE
Is there a roadmap for features/supports for upcoming releases?

### ISSUE THREAD

**Issue by Kralizek** [dotnet/aspire#1013]
Title basically.

It would be interesting if you could disclose at least what's going to be in the next preview. Timeline would be gold, but not necessary.

**Comment by mitchdenny** [comment]
We try to use the milestones feature in GitHub to keep track of what is coming in each release: 

https://github.com/dotnet/aspire/milestones

It isn't a solid commitment, but more _aspirational_.

**Comment by josephaw1022** [comment]
I'm seeking clarification and considering the implementation of a project board for our ongoing work. The primary aim is to enhance visibility into our project's progress and to outline future goals. Specifically, I have a few points to discuss:

1. **Purpose of the Project Board**: Is the primary function of the project board to track our current progress, detailing what has been completed and what's pending?

2. **Visibility of High-Level Goals**: Can the board also serve as a platform to display our high-level future objectives? This would include planned additions like new components, dashboard refinements, documentation updates, adding other cloud components, etc.

3. **Example for Reference**: I found an example that closely aligns with our needs. It's a project board from another initiative and provides a good template for what we might aim for. You can view it here: [dotnet/aspire Project Board Example](https://github.com/dotnet/aspire/assets/47674962/be2a17d7-196a-4d62-8588-1d509eaa1679).

To sum up, I'm proposing that we create a project board similar to the example provided, tailored to Aspire specific requirements.

**Comment by Kralizek** [comment]
Thanks @mitchdenny. I looked at the project tab but didn't think to check the milestones.
